src/Creator/OrderCreator.php

```diff
@@ -25,6 +27,12 @@ public function __construct(
         $this->faker = $faker;
         $this->cartCreator = $cartCreator;
         $this->customerCreator = $customerCreator;
+
+        // Because we could be in CLI mode, there might be no employee in context, so we must set it manually
+        $context = Context::getContext();
+        if (!isset($context->employee) || !isset($context->employee->id)) {
+            $context->employee = new Employee(1);
+        }
```

Comment:
```suggestion
```

This service is not related to the context, and even less about its initialisation It shouldn't be its responsibility to init the legacy context

Reply:
We have a useful tool for that, not very well known or used, that allows the Symfony command to initialize the context instead:
https://github.com/PrestaShop/PrestaShop/blob/d68bfe619ea9ef8e9d57e96ce817028c68f3b885/tests/Integration/PrestaShopBundle/Command/LoadLegacyClassesinCommandTest.php#L123-L124

Reply:
@matthieu-rolland friendly reminder 😉

Reply:
@matks when using the tool suggested by @jolelievre I saw that it was broken for this use case, so I made a fix on the core

I need this PR to be merged first: https://github.com/PrestaShop/PrestaShop/pull/36358

Reply:
PR is now fixed 😄 https://github.com/PrestaShop/PrestaShop/pull/36358